ui/core-react/src/core-react/icons/ConditionalIconItem.tsx

```diff
@@ -27,6 +27,16 @@ export class ConditionalIconItem {
     this._value = value;
   }
 
+  /** A work-around for isInstanceOf */
+  public static isConditionalIconItem(item: any): boolean {
```

Comment:
Do we intend to use this internally only, maybe we could mark this as @internal?
Would this make more sense as a type-guard?
Could we avoid `any` type?
Adding a new public API with "workaround" docs is not ideal.

Reply:
Good idea -- i should have marked it internal.